pkg/ccl/backupccl/backuppb/backup.go

```diff
@@ -179,11 +191,220 @@ func (e *ExportStats) Combine(other bulk.TracingAggregatorEvent) {
 	e.Duration += otherExportStats.Duration
 }
 
-// Tag implements the AggregatorEvent interface.
+// Tag implements the TracingAggregatorEvent interface.
 func (e *ExportStats) Tag() string {
 	return "ExportStats"
 }
 
+// Identity implements the TracingAggregatorEvent interface.
+func (s *IngestionPerformanceStats) Identity() bulk.TracingAggregatorEvent {
+	stats := IngestionPerformanceStats{}
+	stats.SendWaitByStore = make(map[roachpb.StoreID]time.Duration)
+	return &stats
+}
+
+// Combine implements the TracingAggregatorEvent interface.
+func (s *IngestionPerformanceStats) Combine(other bulk.TracingAggregatorEvent) {
+	otherStats, ok := other.(*IngestionPerformanceStats)
+	if !ok {
+		panic(fmt.Sprintf("`other` is not of type IngestionPerformanceStats: %T", other))
+	}
+
+	s.DataSize += otherStats.DataSize
+	s.BufferFlushes += otherStats.BufferFlushes
+	s.FlushesDueToSize += otherStats.FlushesDueToSize
+	s.Batches += otherStats.Batches
+	s.BatchesDueToRange += otherStats.BatchesDueToRange
+	s.BatchesDueToSize += otherStats.BatchesDueToSize
+	s.SplitRetries += otherStats.SplitRetries
+	s.Splits += otherStats.Splits
+	s.Scatters += otherStats.Scatters
+	s.ScatterMoved += otherStats.ScatterMoved
+	s.FillWait += otherStats.FillWait
+	s.SortWait += otherStats.SortWait
+	s.FlushWait += otherStats.FlushWait
+	s.BatchWait += otherStats.BatchWait
+	s.SendWait += otherStats.SendWait
+	s.SplitWait += otherStats.SplitWait
+	s.ScatterWait += otherStats.ScatterWait
+	s.CommitWait += otherStats.CommitWait
+	s.Duration += otherStats.Duration
+
+	for k, v := range otherStats.SendWaitByStore {
+		s.SendWaitByStore[k] += v
+	}
+}
+
+// Tag implements the TracingAggregatorEvent interface.
+func (s *IngestionPerformanceStats) Tag() string {
+	return "IngestionPerformanceStats"
+}
+
+// Render implements the TracingAggregatorEvent interface.
+func (s *IngestionPerformanceStats) Render() []attribute.KeyValue {
+	const mb = 1 << 20
+	tags := make([]attribute.KeyValue, 0)
+	if s.Batches > 0 {
+		tags = append(tags,
+			attribute.KeyValue{
+				Key:   tagNumBatches,
+				Value: attribute.Int64Value(s.Batches),
+			},
+			attribute.KeyValue{
+				Key:   tagNumBatchesDueToSize,
+				Value: attribute.Int64Value(s.BatchesDueToSize),
+			},
+			attribute.KeyValue{
+				Key:   tagNumBatchesDueToRange,
+				Value: attribute.Int64Value(s.BatchesDueToRange),
+			},
+			attribute.KeyValue{
+				Key:   "split_retires",
```

Comment:
nit: lets define all strings as top level consts? Or inline all the strings for consistency.